deploy/docker/Dockerfile

```diff
@@ -1,28 +1,44 @@
 FROM alpine:3.5
 
-LABEL maintainer="adundovi@gmail.com"
-LABEL description="Prepares a docker image \
-of pump.io server, http://pump.io/."
+LABEL maintainer Andrej Dundovic <adundovi@gmail.com>
+LABEL description "Stream server that does \
+most of what people really want from a social network."
 
 ENV LC_ALL en_US.UTF-8
 
+ENV PUMPIO_VERSION 3.0.2
+
 RUN apk add --no-cache \
-  nodejs \
-  git \
-  python \
-  make \
-  g++ \
-  graphicsmagick
-RUN npm install -g databank-mongodb
-RUN npm install -g pump.io
-RUN adduser -D -g "" pumpio
+    curl \
+    nodejs \
+    python \
+    make \
+    g++ \
+    git \
+    graphicsmagick \
+ && adduser -D -g "" pumpio \
+ && mkdir -p /etc/ssl/private \
+ && npm install -g databank-mongodb@0.19.2 \
+ && mkdir -p /usr/src \
+ && curl -fSL https://github.com/pump-io/pump.io/archive/v$PUMPIO_VERSION.zip -o pumpio.zip \
```

Comment:
Wait wait wait. I thought we decided to only focus on the production Docker image in this PR, and split the dev environment off into a separate issue?

> When Dockerfile is copying and adding files from the outside of its own dir, it doesn't inspire confidence...

I just don't see how this is a problem. Who cares about the difference between `.` and `../..`? If the Dockerfile was touching things outside the _repository clone_, say, `~/.local`, _that_ would be super sketchy. But I just don't see the argument that using things outside of the _directory_ is bad. Again, anyone using the Dockerfile will just be cloning the repository anyway. This feels a little like [cargo-cult programming][].

 [cargo-cult programming]: https://en.wikipedia.org/wiki/Cargo_cult_programming